subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -1391,7 +1402,9 @@ int nrf_cloud_pgps_process(const char *buf, size_t buf_len)
 		LOG_ERR("P-GPS subsystem is not initialized.");
 		return -EINVAL;
 	}
-	LOG_HEXDUMP_DBG(buf, buf_len, "MQTT packet");
+
+	LOG_HEXDUMP_DBG(buf, buf_len, "P-GPS data packet");
```

Comment:
this can be removed, the payload is json.
also the comment `/* handle incoming MQTT packets */`  at the top of the function should be edited/removed.

Reply:
Thanks, removed both comments